cannot transmute between types of different sizes, or dependently-sized types

Not the first time this is mentioned (#191). On the latest master, I get the following error:

```bash
error[E0512]: cannot transmute between types of different sizes, or dependently-sized types
  --> /home/anon/.cargo/registry/src/github.com-1ecc6299db9ec823/sp-runtime-interface-5.0.0/src/impls.rs:45:1
   |
45 | assert_eq_size!(usize, u32);
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: source type: `usize` (64 bits)
   = note: target type: `u32` (32 bits)
   = note: this error originates in the macro `assert_eq_size` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0512]: cannot transmute between types of different sizes, or dependently-sized types
  --> /home/anon/.cargo/registry/src/github.com-1ecc6299db9ec823/sp-runtime-interface-5.0.0/src/impls.rs:47:1
   |
47 | assert_eq_size!(*const u8, u32);
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: source type: `*const u8` (64 bits)
   = note: target type: `u32` (32 bits)
   = note: this error originates in the macro `assert_eq_size` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0512`.
error: could not compile `sp-runtime-interface` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
error: build failed
```

I can fix this by explicitly adding the `sp-runtime` sub-dependency (which I don't directly need) to my dependency list in `Cargo.toml` and enabling the `"std"` feature.

```toml
sp-runtime = { version = "5.0.0", default-features = false, features = ["std"] }
```

--- a/subxt/Cargo.toml
+++ b/subxt/Cargo.toml
@@ -30,7 +30,7 @@ thiserror = "1.0.24"
 subxt-macro = { version = "0.17.0", path = "../macro" }
 
 sp-core = { version = "5.0.0", default-features = false  }
-sp-runtime = { version = "5.0.0", default-features = false, features = ["std"] }
+sp-runtime = "5.0.0"
 sp-version = "4.0.0"
 
 frame-metadata = "14.0.0"

--- a/subxt/Cargo.toml
+++ b/subxt/Cargo.toml
@@ -30,7 +30,7 @@ thiserror = "1.0.24"
 subxt-macro = { version = "0.17.0", path = "../macro" }
 
 sp-core = { version = "5.0.0", default-features = false  }
-sp-runtime = { version = "5.0.0", default-features = false }
+sp-runtime = { version = "5.0.0", default-features = false, features = ["std"] }
 sp-version = "4.0.0"
 
 frame-metadata = "14.0.0"

